pkg/resources/convertible.go

```diff
@@ -29,14 +29,17 @@ func ConvertibleResources() ([]ConvertibleResourceDef, error) {
 	p := provider.MezmoProvider{}
 	terraformRes := p.Resources(context.Background())
 	convertibleRes := make([]ConvertibleResourceDef, len(terraformRes))
+	var keys []string
 	for i, tfResFn := range terraformRes {
 		tfRes := tfResFn()
 		cRes, ok := tfRes.(ConvertibleResourceDef)
+		keys = append(keys, cRes.NodeType())
 		if !ok {
 			err := fmt.Errorf("terraform resource %T does not implement ConvertibleResourceDef", tfRes)
 			return nil, err
 		}
 		convertibleRes[i] = cRes
 	}
+	fmt.Println(keys)
```

Comment:
Intentional?

Reply:
Nope. Dumped keys out for Terry and got bundled here.